h2o-py/h2o/h2o.py

```diff
@@ -766,6 +770,7 @@ def parse_setup(raw_frames, destination_frame=None, header=0, separator=None, co
     :param custom_non_data_line_markers: If a line in imported file starts with any character in given string it will NOT be imported. Empty string means all lines are imported, None means that default behaviour for given format will be used
     :param partition_by: A list of columns the dataset has been partitioned by. None by default.
     :param quotechar: A hint for the parser which character to expect as quoting character. Only single quote, double quote or None (default) are allowed. None means automatic detection.
+    :param escape_char: One-character string used to escape other characters. '\' by default.
```

Comment:
Now that this is exposed, I'm wondering if it should not be None by default (char=0 on Java side) to be in sync with pandas.

Reply:
can be, but then there will be needed changes everywhere in tests where we are parsing something escaped by the escapechar which was hardcoded in h2o until now ('\\')

Reply:
correct, but it's only 2 or 3 of them: the ones I created in this PR: https://github.com/h2oai/h2o-3/pull/5388/files